packages/types/src/cycle/cycle.d.ts

```diff
@@ -31,7 +31,7 @@ export interface ICycle {
   unstarted_issues: number;
   updated_at: Date;
   updated_by: string;
-  archived_at: string | null;
```

Comment:
This is coming form an API, it can never be a Date, it will be of type string.

Reply:
okay.. Fixing all of them.